domain/src/test/java/com/example/domain/policy/MovieDayDiscountPolicyTest.kt

```diff
@@ -1,16 +1,28 @@
-package woowacourse.movie.domain.policy
+package com.example.domain.policy
 
 import junit.framework.TestCase
 import org.junit.Test
 import java.time.LocalDateTime
 
 class MovieDayDiscountPolicyTest {
     @Test
-    fun `무비데이라면 true를 반환한다`() {
+    fun `10일이라면 true를 반환한다`() {
```

Comment:
```suggestion
    fun `10일이라면 무비데이이다`() {
```
지금도 잘 작성해주셨지만, "무비데이"라는 단어를 언급해주면 더 좋을 것 같아요. (이하 관련 내용 동일)